src/definitions/behaviors/form.js

```diff
@@ -1823,6 +1824,58 @@ $.fn.form.settings = {
       ;
     },
 
+    // applies rule if another field's value satisfies a condition
+    condition: function(value, ancillary, $module) {
+      let params = ancillary.split(',');
+      if(params.length !== 3) {
+        return false;
+      }
+
+      let
+          identifier  = params[0].trim(),
+          condition   = params[1].trim(),
+          rule        = params[2].trim()
+      ;
+      if(rule.includes('condition')) {
```

Comment:
Please use `indexOf('condition') >-1` instead as we are still supporting IE11

Reply:
Done
Updated PR and JSFiddle. I tested with a n IE11 emulator, it looks like it's working. 
Please let me know if I missed anything